stdlib/universe/universe.flux

```diff
@@ -205,12 +206,19 @@ cov = (x, y, on, pearsonr=false) => join(
     |> covariance(pearsonr: pearsonr, columns: ["_value_x", "_value_y"])
 pearsonr = (x, y, on) => cov(x: x, y: y, on: on, pearsonr: true)
 
-// AggregateWindow applies an aggregate function to fixed windows of time.
+_fillEmpty = (tables=<-, createEmpty) => if createEmpty then
+    tables
+        |> table.fill()
+else
+    tables
+
```

Comment:
This is tangential to your changes, but I just wanted to comment on the formatting here, which looks odd to me. I think I would prefer something like:

```suggestion
_fillEmpty = (tables=<-, createEmpty) => if createEmpty then
    tables |> table.fill()
else
    tables
```

But obviously that's not your responsibility for this PR. Just wanted to make note of it in case we wanted to file an issue for it later.